packages/terra-form-select/src/shared/_MenuUtil.js

```diff
@@ -218,7 +221,10 @@ class MenuUtil {
    */
   static findPrevious(object, value) {
     const options = MenuUtil.flatten(object, true);
-    const index = options.findIndex(({ props }) => MenuUtil.isEqual(props.value, value));
+    let index = options.findIndex(({ props }) => MenuUtil.isEqual(props.value, value));
+    if (index === 0) {
+      index = options.length;
+    }
     return index === -1 ? null : options[Math.max(index - 1, 0)].props.value;
```

Comment:
If we are checking the index in the preceding if block, we should no longer require the Math.max function, correct?

Reply:
Removed Math.max function